docs/source/usage_guides/deepspeed.mdx

```diff
@@ -28,16 +29,16 @@ You can find more information about DeepSpeed Gaudi integration [here](https://d
 
 ## Setup
 
-To use DeepSpeed on Gaudi, you need to install Optimum Habana and [Habana's DeepSpeed fork](https://github.com/HabanaAI/DeepSpeed) with:
+To use DeepSpeed on Gaudi, you need to install Optimum for Intel Gaudi and [Habana's DeepSpeed fork](https://github.com/HabanaAI/DeepSpeed) with:
```

Comment:
To use DeepSpeed on Gaudi, you need to install Optimum for Intel Gaudi and [DeepSpeed fork]

Reply:
Done